academic-services/graph/reference-data-schema.md

```diff
@@ -40,9 +40,10 @@ Column # | Name | Type | Note
 8 | PaperCount | long |
 9 | PaperFamilyCount | long | See [FAQ](resources-faq.md#papercount-vs-paperfamilycount)
 10 | CitationCount | long |
-11 | Latitude | float? |
-12 | Longitude | float? |
-13 | CreatedDate | DateTime |
+11 | Iso3166Code | string | Two-letter codes (alpha-2) defined in [ISO_3166 Code](https://www.iso.org/iso-3166-country-codes.html) (ISO.org) and [ISO 3166-2](https://en.wikipedia.org/wiki/ISO_3166-2) (Wiki)
```

Comment:
Suggestion: Put an example (en-US), as a lot of folks have no idea what this is.

Reply:
example added.